src/api/app/views/webui2/webui/project/_pulse_list.html.haml

```diff
@@ -0,0 +1,91 @@
+.row
+  .col
+    %p
+      During this period
+      = render partial: 'pulse_list_commits'
+    %p
+      = render partial: 'pulse_list_builds'
+      = render partial: 'pulse_list_new'
+      = render partial: 'pulse_list_changes'
+    %p
+      = render partial: 'pulse_list_branches'
+      = render partial: 'pulse_list_comments'
+
+  .col
+    .card
+      .card-header
+        Requests
+      .card-body
+        - if @requests.any?
+          .row
+            .col
+              .progress
+                -# haml-lint:disable InlineStyles
```

Comment:
Why don't you use `width` instead of `style` and then you don't need to disable haml?

Reply:
HTML width is not CSS width. Tried it, HTML width doesn't work on div's.